components/ModDashboardLeftNavbar.vue

```diff
@@ -88,4 +90,10 @@ function createCountsForSubmissionListView(submissions: Submission[]) {
 }
 
 const autofillStatusCount = computed(() => createCountsForSubmissionListView(moderationSubmissionsStore.submissionsData))
+
+watchEffect(() => {
+    if (moderationSubmissionsStore.submissionsData.length > 0) {
```

Comment:
Style option: you can remove the >0 part and most people can read this fine

```suggestion
    if (moderationSubmissionsStore.submissionsData.length) {
```